layouts/topics/kubernetes.html

```diff
@@ -1,9 +1,11 @@
 {{ define "hero" }}
     <header class="header-hero header-hero-glow">
         <div class="header-hero-items container mx-auto">
-            <div class="header-hero-item">
+            <div class="header-hero-item pr-20">
```

Comment:
I don't think this pr-20 is being applied (because of the properties of header-hero-item).